galsim/config/process.py

```diff
@@ -20,8 +20,20 @@
 import galsim
 import logging
 
-from .input import *
-from .output import *
+
+# Python 2.6 doesn't include OrderdDict natively.  There is a package ordereddict that you
+# can pip install.  But if the user hasn't done that, we'll just read into a regular dict.
+# The only feature that requires the OrderedDict is the truth catalog output.  With a regular
+# dict the columns will appear in arbitrary order.
```

Comment:
Should galsim emit some kind of warning in the case that this is going to happen (i.e., if we use a regular dict instead of an ordered dict)?